src/dmd/toobj.d

```diff
@@ -521,6 +521,9 @@ void toObjFile(Dsymbol ds, bool multiobj)
                 dtb.size(0);
 
             // m_RTInfo
+            if(!cd.getRTInfo)
+                error(cd.loc, "ICE: RTInfo not evaluated for %s", cd.toChars());
```

Comment:
I don't care for the use of "ICE" in the error message; it's too compiler-jargony and seems redundant.  Why isn't a generic error sufficient.  i.e. why not simply print "\`RTInfo\` was not evaluated for \`%s\`"?  Or, maybe this should be an `assert` instead?

P.S.  Don't forget the tick marks.

Reply:
I've removed the ICE errors as it goes against the recent trend: if it's not defined in the library, don't emit it.